app/workers/cassandra_writer.rb

```diff
@@ -0,0 +1,8 @@
+class CassandraWriter
+  include Sneakers::Worker
+  from_queue "worker.events"
+
+  def work(event_json)
```

Comment:
Unused method argument - `event_json`. If it's necessary, use `_` or `_event_json` as an argument name to indicate that it won't be used. You can also write as `work(*)` if you want the method to accept any arguments but don't care about them.